.editorconfig

```diff
@@ -0,0 +1,17 @@
+# Top-most EditorConfig file
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+charset = utf-8
+
+[*.hbs]
+indent_style = space
+indent_size = 2
+
+[*.sh]
+indent_style = space
+indent_size = 2
```

Comment:
What is this file?

Reply:
it tells your editor how to style the indention of this project

Reply:
Hm, seems like a few editors are supported but many are not.

*shrug* I have no idea if this file here actually reflects my intentions, but I guess I'll complain when I see PRs with the wrong indentation. ;)